examples/snippets/modules/ROOT/examples/build-flink-streamlets-java/step3/src/main/java/com/example/FlinkProcessor.java

```diff
@@ -18,7 +18,7 @@
   //         the partitioner function explicitly or else RoundRobinPartitioner will
   //         be used : using `name` as the partitioner here
   AvroInlet<Data> in = AvroInlet.<Data>create("in", Data.class);
-  AvroOutlet<Data> out = AvroOutlet.<Data>create("out", (Data d) -> d.getId().toString(), Data.class);
+  AvroOutlet<Data> out = AvroOutlet.<Data>create("out", (Data d) -> Integer.toString(d.getId()), Data.class);
```

Comment:
out of curiosity: What's the difference here?

Reply:
Id became a primitive int instead of an Integer, using the new avro libs.